test_homework20_library.py

```diff
@@ -0,0 +1,157 @@
+"""Unittest for Homework 11: library."""
+
+import unittest
+import logging
+from homework11_library import Book, User
+
+formatter = logging.Formatter('[%(asctime)s] %(levelname)s - %(message)s')
+file_handler = logging.FileHandler('test_library.log')
+file_handler.setFormatter(formatter)
+
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(formatter)
+
+logger = logging.getLogger(__name__)
+logger.addHandler(file_handler)
+logger.addHandler(console_handler)
+logger.setLevel(logging.INFO)
```

Comment:
You can use 1 logger across single application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application

Reply:
Done!